.github/workflows/workflows.json

```diff
@@ -24,17 +24,20 @@
   "cloud-tasks/tutorial-gcf/function",
   "composer",
   "composer/functions/composer-storage-trigger",
+  "contact-center-insights",
   "container-analysis/snippets",
   "containerengine/hello-world",
   "container/snippets",
   "datacatalog/cloud-client",
+  "datacatalog/quickstart",
   "datalabeling",
   "datastore/functions",
-  "datacatalog/quickstart",
+  "datastore/functions",
```

Comment:
Nitpick: `datastore/functions` appears twice in this list.

Reply:
Good catch!